core/src/main/java/org/apache/iceberg/hadoop/HadoopTableOperations.java

```diff
@@ -234,7 +254,7 @@ public long newSnapshotId() {
   }
 
   @VisibleForTesting
-  Path getMetadataFile(int metadataVersion) throws IOException {
+  Path getMetadataFile(Integer metadataVersion) throws IOException {
```

Comment:
is "null" valid to pass here?

Reply:
Because when Mokito proxy executes a method, the parameter cannot be a basic type, otherwise it will report an error.

Reply:
I'm sorry. It's my fault. I'll put it back.